website/docs/r/container_app_job.html.markdown

```diff
@@ -314,9 +314,19 @@ A `volume` block supports the following:
 
 A `secret` block supports the following:
 
-* `name` - (required) Name of the secret.
+* `identity` - (Optional) The identity to use for accessing the Key Vault secret reference. This can either be the Resource ID of a User Assigned Identity, or `System` for the System Assigned Identity.
 
-* `value` - (required) Value of the secret.
+!> **Note:** `identity` must be used together with `key_vault_secret_id`
+
+* `key_vault_secret_id` - (Optional) The ID of a Key Vault secret. This can be a versioned or version-less ID.
+
+!> **Note:** When using `key_vault_secret_id`, `ignore_changes` should be used to ignore any changes to `value`.
+
+* `name` - (Required) The secret name.
```

Comment:
Can we have this back at the top of the block as it's a required property? (this takes precedence over alphabetical ordering)

Reply:
Sure thing. I got this from `container_app` so I will reorder it there as well, then.